### PR TITLE
Split EQUIPMENT_MASS_COEFFICIENTS to 13 individual variables

### PR DESCRIPTION
**Pull request by xjjiang**
### Summary

This PR splits EQUIPMENT_MASS_COEFFICIENTS to 13 individual variables. They are (in the same order):

- Aircraft.APU.MASS
- Aircraft.Instruments.MASS
- Aircraft.Hydraulics.FC_MASS_COEFFICIENT
- Aircraft.Hydraulics.GEAR_MASS_COEFFICIENT
- Aircraft.Avionics.MASS
- Aircraft.AirConditioning.MASS_COEFFICIENT
- Aircraft.AntiIcing.MASS
- Aircraft.Furnishings.MASS
- Aircraft.CrewPayload.PASSENGER_SERVICE_MASS_PER_PASSENGER
- Aircraft.CrewPayload.WATER_MASS_PER_OCCUPANT
- Aircraft.Design.EMERGENCY_MASS
- Aircraft.CrewPayload.CATERING_ITEMS_MASS_PER_PASSENGER
- Aircraft.Fuel.UNUSABLE_FUEL_MASS_COEFFICIENT

All of them have units "unitless" or "lbm". Once this change is merged to Aviary, we can claim that all Aviary variables have appropriate units.

These variables correspond to `CW(1), ..., CW(13)` in GASP. 

Note that `CW(5)` has two different units (lb or lb/pax) in the new version of GASP. In the above, I only consider one unit: `lbm`.

Also note that the new version of GASP has extended `CW` array to 16 entries. `CW(15)` is the electrical Systems weight per passenger. Aviary has this weight in `equipment_and_useful_load.py` already. So, I suggest a new variable:

ELECTRICAL_MASS_PER_PASSENGER: mass of electrical system per passenger (lbm)

`CW(14)` is cargo handling. CW(16) is other operating items. They both default to 0.0. Aviary does not have them. We can add them later.

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Comment by xjjiang** [comment]
Wait for me. I will reuse variables from FLOPS.

**Comment by xjjiang** [comment]
@crecine and @johnjasa I modified those 13 new variables so that they fall into Aviary categories. The only exception is `Aircraft.Design.EMERGENCY_MASS` which I am not sure where it belongs to.

File: aviary/variable_info/variable_meta_data.py [hunk 1]
```diff
@@ -972,6 +1020,18 @@
     desc='Drag polar computed during Aviary pre-mission.',
 )
 
+add_meta_data(
+    Aircraft.Design.EMERGENCY_MASS,
```
Comment (jkirk5) [inline aviary/variable_info/variable_meta_data.py:1024]: @crecine does GASP have any additional information on what's included in emergency mass? Design is an ok place to put it if it doesn't neatly fit into any of our other categories

File: aviary/variable_info/variable_meta_data.py [hunk 2]
```diff
@@ -3374,6 +3434,30 @@
 #           |___/
 # ====================================================================
 
+add_meta_data(
+    Aircraft.Hydraulics.FC_MASS_COEFFICIENT,
```
Comment (jkirk5) [inline aviary/variable_info/variable_meta_data.py:3438]:
```suggestion
    Aircraft.Hydraulics.FLIGHT_CONTROL_MASS_COEFFICIENT,
```

**Review by jkirk5 — APPROVED** [review]

File: aviary/variable_info/variable_meta_data.py (hunk 2 above)
Comment (jkirk5) [reply aviary/variable_info/variable_meta_data.py:3438]: If this variable gets changed, the new name needs to get updated throughout the code

File: aviary/variable_info/variable_meta_data.py (hunk 1 above)
Comment (crecine) [reply aviary/variable_info/variable_meta_data.py:1024]: I'll see if I can find any more detailed information, but it's the cabin emergency equipment (fire extinguishers, oxygen tanks, smoke hoods, etc)

File: aviary/utils/fortran_to_aviary.py [hunk 3]
```diff
@@ -247,13 +246,15 @@ def process_and_store_data(data, var_name, legacy_code, current_namelist, altern
         var_values = []
 
     if '(' in var_name:  # some GASP lists are given as individual elements
-        # get the target index (Fortran uses 1 indexing, Python uses 0 indexing)
-        fortran_offset = 1 if current_namelist else 0
-        var_ind = int(var_name.split('(')[1].split(')')[0])-fortran_offset
-        var_name = var_name.split('(')[0]  # remove the index formatting
+        if not var_name.startswith("CW("):
```
Comment (crecine) [inline aviary/utils/fortran_to_aviary.py]: generalize special handling for CW

**Review by crecine — APPROVED** [review]

**Pull request merged**